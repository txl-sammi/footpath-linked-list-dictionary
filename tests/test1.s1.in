Palmerston Street between Rathdowne Street and Drummond Street
