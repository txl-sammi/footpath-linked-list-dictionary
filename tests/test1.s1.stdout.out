Palmerston Street between Rathdowne Street and Drummond Street --> 1
